superset/migrations/versions/2023-03-17_13-24_b5ea9d343307_bar_chart_stack_options.py

```diff
@@ -0,0 +1,95 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""bar_chart_stack_options
+
+Revision ID: b5ea9d343307
+Revises: d0ac08bb5b83
+Create Date: 2023-03-17 13:24:54.662754
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "b5ea9d343307"
+down_revision = "d0ac08bb5b83"
+
+import json
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy import and_, Column, Integer, String, Text
+from sqlalchemy.ext.declarative import declarative_base
+
+from superset import db
+
+Base = declarative_base()
+
+CHART_TYPE = "echarts_timeseries_bar"
```

Comment:
shouldn't we also migrate the other ECharts "timeseries" like plugins to use the new select-based control? Here I'm using the ECharts "Generic Chart", and the stack control is still a checkbox:
<img width="806" alt="image" src="https://user-images.githubusercontent.com/33317356/225912090-ec9e5bf2-70f4-4aa2-8cd4-852dccb3c262.png">